### PR TITLE
fix(cli): legacy pin api, empty 200 on add to support 3id-connect, ot…

### PR DESCRIPTION
**Pull request by zachferland**
Related context - https://discord.com/channels/484729862368526356/1075891586354925708/1079555831537881198

get /pins/streamid - throw error
get /pins/ - throw error 
post /pins/streamid - ~~empty 200~~ 200 with warning message, not pinned, req ignored/dropped
delete /pins/streamid - throw error

Not ideal as it doesnt communicate expected behavior, but worthwhile temporary tradeoff (ref discord). Likely less effect on pin.add as well, since streams are pinned by default on creation, people calling pin.add are likely calling it on stream already pinned, while rm/list still throw errors (and more likely to be used by node runners). With plan to change with a stable path after.

### CODE REVIEW

File: packages/cli/src/ceramic-daemon.ts [hunk 1]
```diff
@@ -968,6 +972,10 @@ export class CeramicDaemon {
       .json({ error: 'Method not supported: pin requests have moved to the admin API /admin/pins' })
   }
 
+  async _empty200(req: Request, res: Response): Promise<void> {
+    res.sendStatus(StatusCodes.OK)
```
Comment (stbrody) [inline packages/cli/src/ceramic-daemon.ts:976]: could we maybe still include a field in the response object with a warning message that this is deprecated and not doing anything?  People using tools are unlikely to see it, but at least if someone manually hits the pin endpoint they'll see something in the response object telling them what's up

Comment (zachferland) [reply packages/cli/src/ceramic-daemon.ts:976]: yeah that makes sense, adding now

**Comment by zachferland** [comment]
Added message, in reference to comment

**Review by stbrody — APPROVED** [review]
lgtm

**Review by oed — APPROVED** [review]

**Pull request merged**